gateway/src/apicast/cli/environment.lua

```diff
@@ -0,0 +1,129 @@
+local pl_path = require('pl.path')
```

Comment:
Would be good to document the purpose of this module.

Reply:
Yep, Opened https://github.com/3scale/apicast/pull/500 so we can see the documentation artifacts and verify it looks correctly. Will write it after that one is merged and I can see the documetation.

Reply:
Documented in https://github.com/3scale/apicast/commit/5aa2e0c985aed253c5b29dba54a8a112d240e5c2